data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1338 @@ mission "Unfettered Jump Drive Trading"
 			choice
-				`	"I think it is true. The Hai who gave this tribute said they hoped the Unfettered would turn from violence and be reconciled with them."`
-					goto true
-				`	"I'm sorry, but I don't think they will ever let you live among them, if it is your nature to be prone to violence."`
-			`	You have little experience reading Hai emotion, but the youth seems crestfallen. "You are certain?" he asks.`
+				`	"Seems easy. I'll do it."`
+					goto agree
+				`	"Why the sudden interest for that ship?"`
+					goto design
+				`	"You could just ask the Hai that come to you from their space about it."`
+			`	"That is what we will be forced to do if you refuse, but we would prefer this to be kept a secret. So do you agree to do this for us?"`
 			choice
-				`	"I am certain. I can do nothing for you."`
-					goto refuse
-				`	"Perhaps you can come with me and find out, but don't get your hopes up."`
-					goto end
-			label true
-			`	"Will you take me to Hai-home?" he asks. "You are able to be outside Unfettered places. Can I go with you?"`
+				`	"It seems easy enough. I'll do it."`
+					goto agree
+				`	"How do I scan it?"`
+					goto scan
+				`	"I don't want to get in trouble with the Hai."`
+			`	"That should not be a problem, the Hai will not bothered about you scanning their ship, if they even notice it."`
+			label design
+			`	"We would like to know how they avoided vulnerabilities in the hull of a ship of such size. Extra space on a ship cannot only be used for cargo... I will tell you more about it later, once you bring us the scans. That is, if you are willing of course."`
 			choice
-				`	"Yes, I will take you there."`
-					goto end
-				`	"Sorry, I can't risk angering the Unfettered by carrying a defector."`
-			label refuse
-			`	Disappointed, the youth leaves. You hope you made the right choice.`
+				`	"Sure, will do."`
+					goto agree
+				`	"How do I scan it?"`
+			label scan
+			`	"You need to use one of our Value Detectors, the information provided by it should be sufficient. If you don't have one, you can install one on Darkcloak, on the way there. You may want to get multiple scanners so you can scan the ship from a longer range, and faster. Please do actively scan it. Passive intel gathered on the fly will not do."`
+			choice
+				`	"Sure, will do."`
+					goto agree
+				`	"I'm not interested in scanning ships for you."`
+					decline
+			label agree
+			`	On the way back to your ship, Mera runs up to you. "One... last... thing, captain!" You let her catch her breath. "We have noticed one Geocoris just entered <waypoints> as expected, but seems to have some special modifications. Please scan that one specifically. We have marked its signature for you to recognize it. We want to have a detailed scan of its cargo. Also, I've just found a captain willing to give you one of our scanners. It will be placed in your cargo hold if there is no space on your ship."`
```

Comment:
```suggestion
			`	On the way back to your ship, Mera runs up to you. "One... last... thing, captain!" You let her catch her breath. "We have noticed one Geocoris just entered <waypoints> as expected, but seems to have some special modifications. Please scan that one specifically. We have marked its signature for you to recognize it. We want to have a detailed scan of its cargo. Also, I've just found a captain willing to give you one of our scanners. It will be placed in your cargo hold if there is no space on your ship.`
```
Also drop this one with the new stuff added.